Update document of Range#to_s (#46230)
rails/rails: guides/source/active_support_core_extensions.md

Extensions to `Range`
---------------------

### `to_fs`

> Active Support defines `Range#to_fs` instead of `to_s` so that it understands an optional format argument. As of this writing the only supported non-default format is `:db`:

Reviewer: ```suggestion
Active Support defines `Range#to_fs` as an alternative to `to_s` that understands an optional format argument. As of this writing the only supported non-default format is `:db`:
```

A small suggestion on the phrasing, but other than that this looks good to me!
